README.md

```diff
@@ -0,0 +1,42 @@
+# Jazz Development Environment
+## Introduction
+Jazz Development Environment provides you with tooling for developing RTC Plugins. Main features include running a jetty server for hot deployment and live debugging of jazz plugins.
+
+Setup and usage is handled in a convention over configuration style, which keeps editing files and dealing with setup to a minimum. It is mainly focused on how jazz-community plugins are developed, but all related tools are open sourced under the [jazz community](https://github.com/jazz-community/) organization.
+
+For a more detailed tour of the features, please see the [wiki](link to features)
```

Comment:
Link is missing.

Should it be this one: https://github.com/jazz-community/jazz-debug-environment/wiki/3.2-Using-the-command-line ?

Reply:
I'm not even sure anymore :grin: But I think we can just leave it out, as the wiki ToC is in the readme anyway.